assets/scss/modules/_notifications.scss

```diff
@@ -47,13 +47,25 @@
 }
 
 .alert {
-  border-left: 4px solid #df3034;
+  border-left: 5px solid #df3034;
   clear: both;
   margin: 0 0 em(20);
-  padding: em(20);
+  padding: em(17) em(15) em(17) em(15); // specific values to match design, or should we always round off?
```

Comment:
we don't round off if it borks the designs, but you could shorthand it:
`padding: em(17) em(15);`